Makefile

```diff
@@ -3,9 +3,13 @@ ENV_DIR := $(shell pwd)/_env
 PYTHON_BIN := $(shell which python)
 
 COMPILER_FLAGS := LIBRARY_PATH=. CC="${CC} -Icpp-common/include"
+CLEAN_SRC_DIR = .
 
-FLAKE8 := ${ENV_DIR}/bin/flake8
-
+# We have not written UTs for a number of modules that do not justify it.   Exclude them from coverage results.
+COVERAGE_EXCL = **/test/**,metaswitch/common/alarms_writer.py,metaswitch/common/alarms_to_dita.py,metaswitch/common/alarms_to_csv.py,metaswitch/common/stats_to_dita.py,metaswitch/common/generate_stats_csv.py,metaswitch/common/mib.py
```

Comment:
Long line - can we break it up onto multiple?

Reply:
not easily.

Backslashes in a makefile get replaced with a single space, but we can't have spaces in this string.

We could whitespace separate it and replace the whitespace with commas, but that's even more fiddly because commas and spaces are special in make, so we'd need to do something even more confusing. I'm thinking it's better to have a long line